scene/3d/visual_instance.cpp

```diff
@@ -180,6 +180,24 @@ bool VisualInstance::get_layer_mask_bit(int p_layer) const {
 	return (layers & (1 << p_layer));
 }
 
+void VisualInstance::set_sorting_offset(float p_offset) {
+	sorting_offset = p_offset;
+	VisualServer::get_singleton()->instance_set_pivot_data(instance, sorting_offset, sorting_use_aabb_center);
+}
+
+float VisualInstance::get_sorting_offset() {
+	return sorting_offset;
+}
+
+void VisualInstance::set_sorting_use_aabb_center(bool p_sort_with_aabb) {
+	VisualServer::get_singleton()->instance_set_pivot_data(instance, sorting_offset, sorting_use_aabb_center);
+	sorting_use_aabb_center = p_sort_with_aabb;
```

Comment:
Isn't this wrong? I would expect the VisualServer method to be called *after* setting the value.
I can change it if confirmed as I'm cleaning up some unwanted clang-format changes from this diff.